docs/modules/hivemq.md

```diff
@@ -0,0 +1,239 @@
+# HiveMQ Module
+
+<img src="../modules_logos/hivemq-testcontainer.png" alt="drawing" width="400"/>
+
+Automatic starting HiveMQ docker containers for JUnit4 and JUnit5 tests.
+This enables testing MQTT client applications and integration testing of custom HiveMQ extensions.
+
+- Community forum: https://community.hivemq.com/
+- HiveMQ website: https://www.hivemq.com/
+- MQTT resources:
+    - [MQTT Essentials](https://www.hivemq.com/mqtt-essentials/)
+    - [MQTT 5 Essentials](https://www.hivemq.com/mqtt-5/)
+    
+Please make sure to check out the hivemq-docs for the [community edition](https://github.com/hivemq/hivemq-community-edition/wiki/) 
+and the [regular edition](https://www.hivemq.com/docs/hivemq/4.7/user-guide/).
```

Comment:
Is it regular edition or enterprise edition?

Reply:
It should be "Community Edition" and "Enterprise Edition" :) 

I corrected it.